code/modules/cm_tech/techs/marine/tier1/points.dm

```diff
@@ -0,0 +1,39 @@
+/datum/tech/repeatable/req_points
+	name = "Requisition Budget Increase"
+	icon_state = "budget_req"
+	desc = "Distributes resources to requisitions for spending."
+
+	announce_name = "ALMAYER SPECIAL ASSETS AUTHORIZED"
+	announce_message = "Additional Supply Budget has been authorised for this operation."
+
+	required_points = 15
+	increase_per_purchase = 1
+
+	flags = TREE_FLAG_MARINE
+	tier = /datum/tier/one
+
+	var/points_to_give = 7000
```

Comment:
That is $700000. Mind the multiplier. No idea what the intended amount is, but it is definitely not _this_.